x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/EsqlTestUtils.java

```diff
@@ -723,10 +724,10 @@ public static Literal randomLiteral(DataType type) {
             case BYTE -> randomByte();
             case SHORT -> randomShort();
             case INTEGER, COUNTER_INTEGER -> randomInt();
-            case UNSIGNED_LONG, LONG, COUNTER_LONG -> randomLong();
+            case LONG, COUNTER_LONG -> randomLong();
```

Comment:
Fixed some random results that didn't seem right. In the end I didn't use this function for testing, so I can revert them back.